niworkflows/interfaces/bids.py

```diff
@@ -248,59 +241,77 @@ class _DerivativesDataSinkInputSpec(DynamicTraitedSpec, BaseInterfaceInputSpec):
     check_hdr = traits.Bool(True, usedefault=True, desc='fix headers of NIfTI outputs')
     compress = traits.Bool(desc="force compression (True) or uncompression (False)"
                                 " of the output file (default: same as input)")
-    desc = Str('', usedefault=True, desc='Label for description field')
```

Comment:
Available BIDS entities are now extracted from the `derivatives.json` config file.